src/components/MainContent/index.js

```diff
@@ -53,13 +55,18 @@ function MainContent() {
                 Submit
             </button>
             <div className={styles.singleCoinDiv}>
-                {/* Use redux favorites slice to indicate if there was no data returned after fetch */}
+                {/* Use redux favorites slice to indicate if there was no data
+                returned after fetch */}
                 {!data?.id ? (
-                    <div>display an error message here - coin is misspelt</div>
+                    <div>Search for a valid coin</div>
                 ) : (
                     <>
                         {data.id}
                         {data.current_price}
+
+                        {/* check if data.id exists in favorites */}
+                        {/* if exists -> display remove from favorites */}
+                        {/* if doesnt exist -> display add to favorites */}
                         <button
```

Comment:
The logic outlined above in the comments is missing here

Reply:
Plan to address this in the next push